snapcraft/tests/unit/commands/test_export_login.py

```diff
@@ -130,5 +130,41 @@ def test_failed_login_with_invalid_credentials(self, mock_login):
 
         self.assertThat(result.exit_code, Equals(1))
         self.assertThat(result.output, Contains(
-            storeapi.constants.INVALID_CREDENTIALS))
+                    storeapi.constants.INVALID_CREDENTIALS))
         self.assertThat(result.output, Contains('Login failed.'))
+
+    @mock.patch.object(storeapi._sca_client.SCAClient,
+                       'get_account_information')
+    @mock.patch.object(storeapi.StoreClient, 'login')
+    @mock.patch.object(storeapi.StoreClient, 'acl')
+    def test_export_with_snap_name(
+            self, mock_acl, mock_login, mock_get_account_information):
+        self.mock_input.return_value = 'user@example.com'
+        mock_acl.return_value = {
+            'snap_ids': None,
+            'channels': ['edge'],
+            'permissions': None,
+        }
+
+        result = self.run_command(['export-login', 'exported'])
```

Comment:
you are not passing the --snaps , so this test doesn't have anything different from test_successful_export, right?

Reply:
when i am trying to pass --snap i get the error Snap 'myapp' was not found in '16' series.

Reply:
what should i do ?

Reply:
@elopio would you prefer a mock, or an update to the fake store?